.travis.yml

```diff
@@ -10,6 +10,12 @@ jobs:
         - make update-props-dryrun
     - stage: unit
       language: java
+      before_install:
+        - sudo pip install --upgrade pip
+        - sudo pip install ansible
+      #  - sudo apt-get update
```

Comment:
Just to clarify: do we really need to have commented out commands in the travis.yml?

Reply:
Thank you. Comments have been removed